coldfront/core/project/views.py

```diff
@@ -269,6 +271,17 @@ def get_context_data(self, **kwargs):
         context['can_request_sec_dir'] = \
             pi_eligible_to_request_secure_dir(self.request.user)
 
+        # show survey responses if available
+        allocation_request = SavioProjectAllocationRequest.objects.filter(
```

Comment:
Only requests with the "Approved - Complete" status should be included. (We don't want to include surveys from denied or pending requests.)

Reply:
Also, let's sort the requests so that earlier ones appear first (Survey 1 is earliest).

`.order_by('request_time')`